scripts/create-dev-data.js

```diff
@@ -640,6 +648,10 @@ async function createData() {
     message: 'Scan successfully completed. See artifact for details.',
     count: 3,
   });
+  localSiteBuildTasks.push({ id: btA11y1.id, type: 'a11y' });
+
+  // write localSiteBuildTasks.json file for viewing out reports
+  await writeFile(localSiteBuildTasksFile, JSON.stringify(localSiteBuildTasks), 'utf-8');
```

Comment:
Is this file present but empty until then? or should it have something in it?

Reply:
You need to make sure your local environment is up to date by running the `yarn create-dev-data` script or the `make everything` script.